docs/feature/storage/indexing-and-storage.md

```diff
@@ -0,0 +1,303 @@
+(indexing-and-storage)=
+(storage-internals)=
+
+# Indexing and storage in CrateDB
+
+:::{article-info}
+---
+avatar: https://avatars.githubusercontent.com/u/7926726?v=4
+avatar-link: https://github.com/marijaselakovic
+avatar-outline: muted
+author: Marija Selakovic
+date: November 12, 2021
+read-time: 8 min read
+class-container: sd-p-2 sd-outline-muted sd-rounded-1
+---
+:::
+
+## Introduction
+
+This article explores the internal workings of the storage layer in CrateDB.
+The storage layer ensures that data is stored in a safe and accurate
+way and returned completely and efficiently.
+CrateDB's storage layer is based on Lucene indexes.
```

Comment:
This documentation fragment expands a bit more into what's inside Lucene on that level. We may add or link to this later somehow. It has been created the other day while curating a more highlevel document about the ingredients of CrateDB, and now might be the right time to backport fragments of it, where applicable.

- https://kotori--148.org.readthedocs.build/en/148/database/cratedb.html#storage-layer